modules/core/include/opencv2/core/cuda/filters.hpp

```diff
@@ -95,6 +96,12 @@ namespace cv { namespace cuda { namespace device
 
             const int x1 = __float2int_rd(x);
             const int y1 = __float2int_rd(y);
+            if (x1 <= NPP_MIN_32S || x1 >= NPP_MAX_32S || y1 <= NPP_MIN_32S || y1 >= NPP_MAX_32S)
+            {
+                elem_type src_reg = src(y1, x1);
```

Comment:
> src(y1, x1)

Doesn't look as valid access anyway.

Reply:
It is valid because it would be handled by BorderReader class.